tests/resources/pyfunc_models/README.md

```diff
@@ -0,0 +1,25 @@
+# Historical Pyfunc Models
```

Comment:
@harupy is this something like what you had in mind?

Reply:
looks great!

Reply:
awesome, i'll rebase and see if the tests pass